tests/mxnet/test_hook_save_scalar_mxnet.py

```diff
@@ -0,0 +1,129 @@
+# Standard Library
+import os
+import time
+from datetime import datetime
+
+# Third Party
+import mxnet as mx
+import pytest
+from mxnet import autograd, gluon, init
+from mxnet.gluon import nn as mxnn
+from tests.core.utils import check_tf_events, delete_local_trials, verify_files
+
+# First Party
+from smdebug.core.modes import ModeKeys
+from smdebug.core.save_config import SaveConfig, SaveConfigMode
+from smdebug.mxnet import Hook as MX_Hook
+
+SMDEBUG_MX_HOOK_TESTS_DIR = "/tmp/test_output/smdebug_mx/tests/"
+
+
+def simple_mx_model(hook, steps=10, register_loss=False, with_timestamp=False):
```

Comment:
steps -> epochs?
also below.

Reply:
The steps is correct in this context.  For this test, we are training the model for few steps only not the entire epoch.